.dependabot/config.yml

```diff
@@ -1953,42 +1953,43 @@ update_configs:
     - match:
         dependency_name: demisto/*
         update_type: all
-        
+
   - package_manager: python
-    directory: /docker/opencti-v4.1.2
+    directory: /docker/xgboost
```

Comment:
@daryakoval something is weird with the changes you are putting in the dependabot config file. Can you please check.

Reply:
git looks at this in strange way.. I just deleted one docker and added another..

Reply:
Instead of deleting previous docker image and add new one... it shifts all docker names down